components/progress/CircularProgress.js

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import './CircularProgress.scss';
+import { classnames } from '../../helpers/component';
+
+const CircularProgress = ({
+    children,
+    className,
+    progress,
+    rootRef,
+    size = 32,
+    backgroundColor = '#555',
+    color = '#4cc75a',
+    ...rest
+}) => {
+    const backgroundCircleStyle = useMemo(
+        () => ({
+            stroke: backgroundColor
+        }),
+        backgroundColor
```

Comment:
I don't think it's expected to pass non-array value as a dependency list. At least judging from typescript definitions it's:

```
type DependencyList = ReadonlyArray<any>;
function useMemo<T>(factory: () => T, deps: DependencyList | undefined): T;
```

Reply:
Yes it needs to be an array 👍